Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1741,9 +1741,9 @@ def lhegensim2018ml(fragment,howMuch):
                                                  digiPremixUp2018Defaults25ns])
 
 steps['DIGIPRMXUP15_PU25']=merge([digiPremixUp2015Defaults25ns])
-steps['DIGIPRMXUP15APVSimu_PU25']=merge([{'--customise': 'SimTracker/SiStripDigitizer/customizeSiStripAPVDynamicGainSimulation.activateSiStripAPVDynamicGain'},digiPremixUp2015Defaults25ns])
+steps['DIGIPRMXUP15APVSimu_PU25']=merge([{'--era': 'Run2_2016_HIPM'},digiPremixUp2015Defaults25ns])
```

Comment:
Same question here on new relvals workflow.

Reply:
@srimanob same comment as above: No, there are new relval workflows that excercise both the new DIGI and the RECO tracking mitigation too, those were defined in #28082. Here I just remove the customization function for the DIGI step in favour of more coincise era.